content/docs/start/model-management/model-registry.md

```diff
@@ -0,0 +1,276 @@
+---
+title: 'Get Started: Model registry'
+description:
+  'Set up a git-based model registry with DVC to track and manage models, their
+  versions and lifecycle stages.'
+---
+
+# Get Started: Model Registry
+
+Just as we use experiment tracking to manage model development, it is a good
+idea to keep a model registry to manage the lifecycle of the models we get from
+our experiments. Using DVC and DVC [Studio](/doc/studio) we will set up a model
+registry where we can discover, share, deploy and audit all our models and which
+will serve as the single source of truth for our model management.
+
+To make use of our model registry, we will need some trained models first. To
+speed things up, we will start from a git
+[repository](https://github.com/iterative/example-get-started-model-management)
+with a model training pipeline already set up and ready to use.
+
+To see how you can track experiments and set up training pipelines with DVC and
+DVCLive, have a look at our getting started guide for
+[experiment management](/doc/start/experiments).
+
+## Adding models
+
+Let's now train a model and add it to the model registry. We will be using
+[DVCLive](/doc/dvclive) and add a model using Python code. This will also
+automatically save the model to DVC.
+
+We use the [`log_artifact`](/doc/dvclive/live/log_artifact) method to save the
+model and add it to the model registry. Open the training script `src/train.py`
```

Comment:
We might also need to clarify that `log_artifact` caches the model with dvc. It also needs to be cached, pushed to remote storage, and the remote storage creds need to be configured in studio.

edit: I see you mention some of this later.

Reply:
I tried to make that a bit more explicit